src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/LicenceMatchingIntegrationTest.kt

```diff
@@ -140,4 +141,35 @@ class LicenceMatchingIntegrationTest : IntegrationTestBase() {
 
     assertThat(result).isEmpty()
   }
+
+  @Test
+  @Sql(
+    "classpath:test_data/clear-all-licences.sql",
+    "classpath:test_data/seed-matching-candidates.sql"
+  )
+  fun `Get licence matches - sort by conditional release date`() {
+    val result = webTestClient.get()
+      .uri("/licence/match?sortBy=conditionalReleaseDate&sortOrder=DESC")
+      .accept(MediaType.APPLICATION_JSON)
+      .headers(setAuthorisation(roles = listOf("ROLE_CVL_ADMIN")))
+      .exchange()
+      .expectStatus().isOk
+      .expectHeader().contentType(MediaType.APPLICATION_JSON)
+      .expectBodyList(LicenceSummary::class.java)
+      .returnResult().responseBody
+
+    assertThat(result?.size).isEqualTo(6)
```

Comment:
You probably don't need this with `containsExactly`...

Reply:
Used containsExactly to verify the order

Reply:
I might be wrong but I thought `containsExactly` would fail if there were more or less results from the extracting... Oh hang on that's AssertJ, not sure it's the same with JUnit 😄